04-onion-routing.md

```diff
@@ -789,9 +789,9 @@ channel.
 ### Requirements
 
 The _erring node_:
-  - SHOULD set `pad` such that the `failure_len` plus `pad_len` is equal to 256.
-    - Note: this value is 118 bytes longer than the longest currently-defined
-    message.
+  - MUST set `pad` such that the `failure_len` plus `pad_len` is at least 256.
```

Comment:
Added MUST item in line with discussion in [spec meeting](https://github.com/lightning/bolts/issues/1033).